src/models/chatbotHistory.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * ChatbotHistory model
+ */
+
+import mongoose from 'mongoose';
+
+const ChatbotHistorySchema = new mongoose.Schema(
+  {
+    data: {
+      type: String,
```

Comment:
This doesn't work. The incoming data is not a string, it's an object. When you run this, you get the following message.

- ValidationError: ChatbotHistory validation failed: data: Cast to String failed

This schema needs to match the request objects shape as described in the documentation that is linked in the story and the example that is also included in the story.

Reply:
done